eksconfig/add-on-node-groups.go

```diff
@@ -68,6 +76,8 @@ type ASG struct {
 	// TODO: handle conflicting flag '--cloud-provider aws'
 	// ref. https://github.com/kubernetes/kubernetes/issues/64659
 	KubeletExtraArgs string `json:"kubelet-extra-args"`
+	// clusterautoscaler per nodegroup
+	ClusterAutoScaler *NGClusterAutoScaler `json:"cluster-autoscaler,omitempty"`
```

Comment:
Let's move this inside `AddOnNodeGroups`

Reply:
talked offline. There is value in having it inside ASG. We can enable/disable CA per nodegroup.